systemtest/pom.xml

```diff
@@ -291,8 +297,35 @@
                             </ignoredUnusedDeclaredDependencies>
                         </configuration>
                     </execution>
+                    <execution>
+                        <phase>generate-sources</phase>
+                        <goals>
+                            <goal>copy-dependencies</goal>
+                        </goals>
+                        <configuration>
+                            <outputDirectory>${project.build.directory}/lib</outputDirectory>
```

Comment:
Why do you need this?

Reply:
Well I did not write the plugin but here is the answer from the `README.md` [1] from repo
```
maven-dependency-plugin is needed for proper loading the classes. Without it the plugin will return NoClassDefFound exception and will fail.
```
Maybe @im-konge  or @Frawless could answer better? 

[1] - https://github.com/skodjob/test-metadata-generator

Reply:
Seems like some bug or something? We do not seem to need it for generating the CRDs or docs.

Reply:
IIRC it's because the way how the test-classes libs are stored during the build etc. I think that we looked on removing the dependency, but it wasn't possible due to something. I can take a look at it, but it was something around the dependencies availability across build phases.